src/main/webapp/resources/js/pages/UserGroupsPage/components/UserGroupMembersTable.jsx

```diff
@@ -0,0 +1,114 @@
+import { UserGroupRole } from "../../../components/roles/UserGroupRole";
```

Comment:
File comments

Reply:
Updated in b8cc896b208449c4d489d83294088ac0b4660d51